src/test/scala/firrtlTests/stage/FirrtlMainSpec.scala

```diff
@@ -324,6 +334,41 @@ class FirrtlMainSpec
       new File(td.buildDir + "/Foo.hi.fir") should (exist)
     }
 
+    Scenario("User compiles to multiple Protocol Buffers") {
+      val f = new FirrtlMainFixture
+      val td = new TargetDirectoryFixture("multi-protobuf")
+      val c = new SimpleFirrtlCircuitFixture
+      val protobufs = Seq("Top.hi.pb", "Child.hi.pb")
+
+      And("some input multi-module FIRRTL IR")
+      val inputFile: Array[String] = {
+        val in = new File(td.dir, c.main)
+        val pw = new PrintWriter(in)
+        pw.write(c.input)
+        pw.close()
+        Array("-i", in.toString)
+      }
+
+      When("the user tries to compile to multiple Protocol Buffers in the target directory")
+      f.stage.main(
+        inputFile ++ Array("-X", "none", "-p", "high", "-td", td.buildDir.toString)
+      )
+
+      protobufs.foreach { f =>
+        Then(s"file '$f' should be emitted")
+        val out = new File(td.buildDir + s"/$f")
+        out should (exist)
+      }
+
+      When("the user compiles the Protobufs to a single High FIRRTL IR")
+      f.stage.main(
+        Array("-I", td.buildDir.toString, "-X", "none", "-E", "high", "-td", td.buildDir.toString, "-o", "Foo")
+      )
+
+      Then("one single High FIRRTL file should be emitted")
+      new File(td.buildDir + "/Foo.hi.fir") should (exist)
+    }
```

Comment:
Can you check that the `.pb` you emit and then read back in is the same? I think if you change the initial emission to `-p chirrtl` (instead of `-p high`) you should be able to just check `==` of the `Circuit` objects.

Reply:
It also would be helpful if the example circuit had an extmodule that we check specifically for because I believe that was broken in an earlier version of this PR.